plotter/heat.go

```diff
@@ -98,6 +98,11 @@ func (h *HeatMap) Plot(c draw.Canvas, plt *plot.Plot) {
 	}
 	// ps scales the palette uniformly across the data range.
 	ps := float64(len(pal)-1) / (h.Max - h.Min)
+	// for some degenerate data sets with h.Max == h.Min, this blows
+	// up and needs to be caught
+	if math.IsNaN(ps) {
```

Comment:
If Min == Max, the plot is meaningless. I would argue the best thing to do here is panic. Maybe this should be more explicit, but this is not the correct answer.